EasyPost/BetaFeatures/Parameters/Trackers/All.cs

```diff
@@ -1,11 +1,13 @@
+using System.Diagnostics.CodeAnalysis;
 using EasyPost.Utilities.Internal.Annotations;
 
 namespace EasyPost.BetaFeatures.Parameters.Trackers
 {
     /// <summary>
     ///     Parameters for <see cref="EasyPost.Services.TrackerService.All"/> API calls.
     /// </summary>
-    public class All : BaseParameters
+    [ExcludeFromCodeCoverage]
```

Comment:
Question: I'd prefer if we could exclude these all via config instead of in-code via this `ExcludeFromCodeCoverage`, is that possible?

Reply:
See comment in PR description. I spent over an hour trying to get it to work, but no dice...